WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModel.swift

```diff
@@ -165,6 +165,10 @@ extension ProductFormViewModel {
         product = EditableProductModel(product: product.product.copy(averageRating: averageRating, ratingCount: ratingCount))
     }
 
+    func updateProductType(productType: ProductType) {
```

Comment:
this doesn't seem to be used in this PR, maybe it can be saved for the PR that implements the product type update?

Reply:
Yes correct, it will be used in the next PR 😉